reporting_app/__init__.py

```diff
@@ -237,26 +235,57 @@ def report_sample(sample_id):
         )
     )
 
+
 @app.route('/charts')
 @flask_login.login_required
 def plotting_report():
     return render_template(
         'charts.html',
         api_url=rest_api().api_url('aggregate/run_elements', paginate=False),
-        ajax_token = get_token()
+        ajax_token=get_token()
     )
 
+
 @app.route('/project_status/')
 @flask_login.login_required
 def project_status_reports():
+    # FIXME: Remove this ugly html generation when the page status becomes more stable
+    from config import project_status as project_status_cfg
```

Comment:
Is this import here for lazy loading, or can it be moved to the top of the file?